### PR TITLE
Resolve pr_check pipeline issues

### PR DESCRIPTION
**Pull request by florkbr**
### Description

Solution borrowed from: https://github.com/RedHatInsights/landing-page-frontend/pull/616

---

### Screenshots

NA

---

### Checklist ☑️
- [x] PR only fixes one issue or story 
- [x] Change reviewed for extraneous code 
- [x] UI best practices adhered to 
- [x] Commits squashed and meaningfully named 
- [x] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [ ] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -20,15 +20,24 @@ IQE_FILTER_EXPRESSION=""
 
 set -exv
 
+CHROME_SHA=$(curl -X GET "https://quay.io/api/v1/repository/cloudservices/insights-chrome-frontend/tag/" | jq .tags\[0\].name -r)
+CHROME_CONTAINER_NAME=chrome-$CHROME_SHA
+
+docker run -d --name $CHROME_CONTAINER_NAME --replace --network bridge quay.io/cloudservices/insights-chrome-frontend:$CHROME_SHA
+
```
Comment (Hyperkid123) [inline pr_check.sh:27]: COpy the index after the run to dist and make sure dist exists

Comment (florkbr) [reply pr_check.sh:29]: @Hyperkid123 that was it!

TL;DR for the curious - this was not required in the landing page repo/`pr_check.sh` as the route to index.html for landing page existed on the proxied env (stage). However for rbac-ui this was not the case - and therefore we get 404's when redirected to `/index.html`.

**Comment by florkbr** [comment]
CC @fhlavac @karelhala. The pipeline is still failing in master - and this resolves the issue for now. Follow up to our discussion in slack here: https://redhat-internal.slack.com/archives/C05M0SNTLM8/p1723133870820329

**Comment by florkbr** [comment]
I'm going to merge this as it now resolves the pipeline issues. We may revert if necessary.
